src/labkey/Experiment.ts

```diff
@@ -272,6 +272,11 @@ export interface LineageOptions extends ExperimentJSONConverterOptions, RequestC
     lsids?: string[]
     /** Include parents in the lineage response. Defaults to true. */
     parents?: boolean
+    /**
+     * Optional material lineage type to filter response -- either "AliquotationOnly" or "DerivationOnly".
+     * Defaults to include all.
+     */
+    materialRunType?: string
```

Comment:
Since this is a highly restricted type consider letting that be explored via the typings. Declare it like this:

```ts
materialRunType?: 'AliquotationOnly' | 'DerivationOnly';
```

nit: Alphabetical (should be placed just above "parents").